DependencyInjection/Configuration.php

```diff
@@ -249,6 +249,7 @@ private function getDbalConnectionsNode(): ArrayNodeDefinition
                     ->cannotBeEmpty()
                     ->defaultValue($this->getDefaultSchemaManagerFactory())
                 ->end()
+                ->scalarNode('result_cache')->end()
```

Comment:
we also need to add this to the XSD schema file, or?

Reply:
yea, think so too, good one :)